src/Network/Network/help/New-AzFirewallPolicySnat.md

```diff
@@ -0,0 +1,142 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# New-AzFirewallPolicySnat
+
+## SYNOPSIS
+Creates SNAT configuration of PrivateRange and AutoLearnPrivateRanges for the firewall policy
+
+## SYNTAX
+
+```
+New-AzFirewallPolicySnat [-PrivateRange <String[]>] [-AutoLearnPrivateRange]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Parameters WhatIf and Confirm should be not included in New-AzFirewallPolicySnat after removed shouldprocess. Please update this doc.

Reply:
@BethanyZhou removed